k8s-charm/src/charm.py

```diff
@@ -73,6 +74,14 @@ def _on_controller_relation_changed(self, event: RelationEvent):
 
         self.unit.status = ActiveStatus()
 
+    def _on_config_changed(self, _):
+        relation = self.model.get_relation("controller")
+        if relation:
+            data = JujuDashData(relation.data[relation.app])
+            dashboard_config, nginx_config = self._render_config(data['controller_url'], data['identity_provider_url'], data['is_juju'])
```

Comment:
not for this pr, but the is_juju thing always confused me..  since in most cases people will be connecting the dashboard to a juju controller.. so is_juju will be set to true.. and for a bool, the default value is False.. so it would make more sense to have an is_jaas variable with the default value of False and JIMM charm would set it to True, while the juju controller charm could leave it unset (with the default of False)

Reply:
also..  _on_controller_relation_changed seems to be doing pretty much the same thing.. so.. shouldn't that function just trigger _on_config_changed and be done with it? no need to have code duplicated in two places